### PR TITLE
Hashtag servlet 

### PR DESCRIPTION
**Pull request by isabellasteains**

### CODE REVIEW

File: src/main/java/codeu/controller/HashtagServlet.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+     private MessageStore messageStore;
+    
+    @Override
+    public void init() throws ServletException {
+        super.init();
+        setMessageStore(MessageStore.getInstance());
+    }
+    
+    
+    /** Sets the messageStore for this Servlet*/
```
Comment (evantyra) [inline src/main/java/codeu/controller/HashtagServlet.java:27]: nit: space after "Servlet"

File: src/main/java/codeu/controller/HashtagServlet.java [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    void setMessageStore(MessageStore messageStore) {
+    this.messageStore = messageStore;
+  }
+    
+ /**
+     * Get the profile page
```
Comment (evantyra) [inline src/main/java/codeu/controller/HashtagServlet.java:33]: Profile page?

File: src/main/java/codeu/controller/HashtagServlet.java [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+        List<Message> messages = messageStore.getHashTagMessages(tag);
+        
+        request.setAttribute("hashtag", tag);
+        request.setAttribute("messages", messages);
+
+
+        request.getRequestDispatcher("/WEB-INF/view/hashtag.jsp").forward(request, response);
+    }
+    /**
+     * Change about me
```
Comment (evantyra) [inline src/main/java/codeu/controller/HashtagServlet.java:54]: Again, profile page :)

File: src/main/java/codeu/controller/HashtagsServlet.java [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import codeu.model.data.Message;
+import codeu.model.store.basic.MessageStore;
+
+public class HashtagsServlet  extends HttpServlet{
```
Comment (evantyra) [inline src/main/java/codeu/controller/HashtagsServlet.java]: Whats the difference between `HashtagsServlet` and `HashtagServlet`?

Could you add a Javadoc describing each of the classes?

File: src/main/webapp/WEB-INF/view/profile.jsp [hunk 5]
```diff
@@ -93,9 +94,7 @@ List<Message> messages = (List<Message>) request.getAttribute("messages");
       for (Message message : messages) {
         String author = UserStore.getInstance()
           .getUser(message.getAuthorId()).getName();
-        String ID = UserStore.getInstance()
-                .getUser(message.getAuthorId()).getId().toString();
-        String URL = "/profile/" + ID;
+        String URL = "/profile/" + author;
```
Comment (evantyra) [inline src/main/webapp/WEB-INF/view/profile.jsp:97]: Why the switch from ID to author?

**Review by JudahBrick — APPROVED** [review]

**Pull request merged**